osu-ac/comparer.py

```diff
@@ -64,12 +65,16 @@ def compare(self, mode):
 
         if(mode == "double"):
             iterator = itertools.product(self.replays1, self.replays2)
+            total = sum(1 for i in itertools.product(self.replays1, self.replays2))
         elif (mode == "single"):
             iterator = itertools.combinations(self.replays1, 2)
+            total = sum(1 for i in itertools.combinations(self.replays1, 2))
         else:
             raise InvalidArgumentsException("`mode` must be one of 'double' or 'single'")
 
-        print("Starting to compare replays")
+        tenth = math.floor(total / 10)
+        done = 0
+        print("Starting a total of {:d} combinations.".format(total))
```

Comment:
can you remove the period here and the other print? none of our other prints have it